tests/issue_1512/run_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+for jj in {1,2,4,8,16,32,64,128,256,512,1024,2048}; do
+    python randomFiles.py $jj
+    echo '----------Number of files: '$jj'----------'
+    echo '---------- Half Meg ----------'
+    echo 'Time'>run_half_$jj.csv
+    for ii in {1..10}; do
+            python transfer_files.py halfMeg wrangler.tacc.utexas.edu <path> >>run_half_$jj.csv 2>>log.log
```

Comment:
I am missing something - what is `<path>` here?  `> >>` looks strange to me, too.  Does this run?